extensions/ql-vscode/src/distribution.ts

```diff
@@ -505,6 +508,10 @@ function codeQlLauncherName(): string {
   return (os.platform() === "win32") ? "codeql.exe" : "codeql";
 }
 
+function alternateCodeQlLauncherName(): string | undefined {
```

Comment:
consider renaming to `deprecatedCodeQlLauncherName()`? Or else at least commenting that that's what this means.

Reply:
ok